system/assets/js/admin/presidecore/preside.confirmation.prompts.js

```diff
@@ -17,15 +18,60 @@
 			}
 		}
 
-		presideBootbox.confirm( $link.data( "confirmationPrompt" ), function( confirmed ) {
-			if ( confirmed ) {
-				if ( $link.get(0).form ) {
-					$( $link.get(0).form ).submit();
-				} else {
-					document.location = $link.attr('href');
+		var $message = $( "<div class=\"form-group\"><label>" + $link.data( "confirmationPrompt" ) + "</label></div>" )
+		  , $input   = $( "<input class=\"bootbox-input form-control\" autocomplete=\"off\" type=\"text\" />" )
+		  , match    = ""
+		;
+
+		if( typeof $link.data( "confirmation-match" ) !== "undefined" ) {
+			match =  $link.data( "confirmation-match" );
+
+			$message
+				.append( "<p class=\"help-block\">Please type <code>" + match + "</code> to proceed.</p>" )
```

Comment:
This should use the i18n system. can do this in js:

```js
i18n.translateResource( "cms:confirmation.prompt.please.type.message", [ "<code>" + match + "</code>" ] );
```

Then add this to `/i18n/cms.properties` file:

```properties
confirmation.prompt.please.type.message=Please type {1} in the input below to proceed.
```

Reply:
I have updated the i18n for message. I also added i18n for confirm and cancel buttons.